docs/index.md

```diff
@@ -37,6 +37,7 @@ Four open source demo applications are included:
 
 Refer to theta-client documentation for the development platform you are using.
 
+* [theta-client documentation](https://ricohapi.github.io/theta-client/index.html)
```

Comment:
Can you add "for versions 1.2 and 1.3" ?

You do not need to add this graphic. But it's cool that it looks like they're intending to build the documentation per version.

<img width="271" alt="Screen Shot 2023-07-19 at 3 33 43 PM" src="https://github.com/theta360developers/theta-client-guide/assets/1075520/74bcb58f-89a8-45b7-a66f-bbd7d18ef0db">